package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"devDependencies": {
-		"@types/node": "9.3.0",
+		"@types/node": "9.4.0",
 		"tslint": "5.9.1",
-		"typescript": "2.6.2"
+		"typescript": "2.8.0"
```

Comment:
Is this meant to test SourceClear somehow, or is that a typo? TypeScript 2.7 only came out today...

Reply:
Sorry - this pull request isn't intended to be merged, I'm testing the integration with source clear to see if it's something that'll be useful to us.